packages/salesforce-adapter/test/filters/email_template_static_files.test.ts

```diff
@@ -0,0 +1,135 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { Element, ElemID, ObjectType, InstanceElement, BuiltinTypes, StaticFile } from '@salto-io/adapter-api'
+import filterCreator from '../../src/filters/email_template_static_files'
+import { FilterWith } from '../../src/filter'
+import { SALESFORCE, EMAIL_TEMPLATE_METADATA_TYPE, METADATA_TYPE } from '../../src/constants'
+import { defaultFilterContext } from '../utils'
+
+describe('emailTemplate static files filter', () => {
+  const SUFFIX = 'suffix'
+  const ATTACHMENTS = 'attachments'
+  const CONTENT = 'content'
+  const FULL_NAME = 'fullName'
+  const ATTACHMENT_AS_STRING = 'attachment'
+  const EMAILCONTENT = 'email-content'
+  const ATTACHMENT_NAME = 'attachment.txt'
+  let elements: Element[]
+
+  const emailTemplateID = new ElemID(SALESFORCE, EMAIL_TEMPLATE_METADATA_TYPE)
+
+  const fields = {
+    [CONTENT]: { refType: BuiltinTypes.STRING },
+    [ATTACHMENTS]: { refType: BuiltinTypes.STRING },
+    [FULL_NAME]: { refType: BuiltinTypes.STRING },
+  }
+
+  const emailType = new ObjectType({
+    annotations: {
+      [METADATA_TYPE]: EMAIL_TEMPLATE_METADATA_TYPE,
+      [SUFFIX]: 'email',
+    },
+    elemID: emailTemplateID,
+    fields,
+    path: ['Objects', 'dir'],
+  })
+
```

Comment:
can use `createMetadataObjectType` to build a proper type with typescript enforcement
```suggestion
  const emailType = createMetadataObjectType({
    annotations: {
      metadataType: 'EMAIL_TEMPLATE_METADATA_TYPE',
      suffix: 'email',
      dirName: 'emails',
    },
    fields,
   })
```
maybe even put it next to the other mock types in `mock_elements.ts`?